packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/common.ts

```diff
@@ -0,0 +1,32 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as logs from '@aws-cdk/aws-logs';
+
+/**
+ * Generic properties for defining a delivery stream destination.
+ */
+export interface DestinationProps {
```

Comment:
Renamed.

I moved it to the destinations module because it's an interface "of convenience" that doesn't need to be used for someone to create an `IDestination`. We will re-use it in our destination implementations simply because they happen to allow the same configuration, but not because they're required to. This is in contrast to `DestinationConfig` and `DestinationBindOptions`, which destinations must use in order to satisfy `IDestination`. Let me know your thoughts